clash-protocols.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version:       2.4
 name:                clash-protocols
-synopsis:            a battery-included library for performant dataflow protocols
+synopsis:            a battery-included library for (dataflow) protocols
```

Comment:
:crying_cat_face:

Reply:
Maybe I should have just replaced it with "slow" :^)